LDMP/gui/DlgCalculateDroughtSummary.ui

```diff
@@ -151,7 +151,7 @@
               </font>
              </property>
              <property name="title">
-              <string>Drought dataset for tier 3</string>
+              <string>Drought dataset for Vulnerability</string>
```

Comment:
Lower case "v"

Reply:
@azvoleff these fixes should be done now, thanks for spotting them